x-pack/solutions/security/plugins/security_solution/server/lib/telemetry/filterlists/prebuilt_rules_alerts.ts

```diff
@@ -49,6 +49,9 @@ export const prebuiltRuleAllowlistFields: AllowlistFields = {
   'kibana.alert.rule.updated_at': true,
   'kibana.alert.rule.uuid': true,
   'kibana.alert.rule.version': true,
+  'kibana.alert.rule.parameters': {
```

Comment:
Is `rule_source` nested under `kibana.alert.rule.parameters` or is it a flat key `kibana.alert.rule.parameters.rule_source`?

Reply:
It's nested under, `kibana.alert.rule.parameters` is the flat key for the whole parameters object but we didn't want to include everything there for document size reasons